### PR TITLE
Fixed zoom test

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**
- Added a fix for failing zoom map test
- Some cleanup of a code

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapsPageObject.java [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.interactivemaps.CreateAMapComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
 
 public class InteractiveMapsPageObject extends ArticlePageObject {
 
+  @FindBy(css = ".wikia-maps-create-map")
+  protected WebElement createMapUnderContribute;
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapsPageObject.java]: protected?

File: src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java [hunk 2]
```diff
@@ -11,16 +10,17 @@
...
 
 /**
  * Set of commonly used actions invoked by executing JavaScript on a web page
  */
 public class JavascriptActions {
 
-  private JavascriptExecutor js;
-  private WebDriver driver;
-  private static int WEBDRIVER_WAIT_TIMEOUT_SEC = 15;
+  private final static int WEBDRIVER_WAIT_TIMEOUT_SEC = 15;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/interactivemaps/PalantirComponentObject.java [hunk 3]
```diff
@@ -1,17 +1,16 @@
 package com.wikia.webdriver.pageobjectsfactory.componentobject.interactivemaps;
 
-import com.wikia.webdriver.common.contentpatterns.PalantirContent;
-import com.wikia.webdriver.common.core.Assertion;
-import com.wikia.webdriver.common.logging.PageObjectLogging;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.special.interactivemaps.InteractiveMapPageObject;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
 
 import org.openqa.selenium.JavascriptExecutor;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/interactivemaps/PalantirComponentObject.java]: MINOR Remove this unused import 'org.openqa.selenium.JavascriptExecutor'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by nikodamn** [comment]
LGTM

**Pull request merged**